app/src/components/Statistics/StatisticsElement.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export const StatisticsElement = ({title, description, icon}) => {
+  return (
+    <Element>
+        <img src={icon} alt="Icon"/>
+        <h2>{title}</h2>
+        <p>{description}</p>
+    </Element>
+  )
+}
+
+const Element = styled.div`
+height: 300px;
+width: 200px;
+border: 1px solid black;
```

Comment:
Please use color from theme object

Reply:
will change px to rem units. border here is for testing. will remove with next PR.

Reply:
ok